jenkins/lava-boot-v2.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*
+#
+
+import os
+from subprocess import Popen
+
+labs = {
+    'lab-free-electrons': {
+        'blacklist': [
+            'android'
+        ],
+    },
+}
+
+# Basic env
+LAB = os.environ['LAB']
+API = os.environ['API']
+API_TOKEN = os.environ['API_TOKEN']
+STORAGE = os.environ['STORAGE']
+CALLBACK = os.environ['CALLBACK']
+GIT_DESCRIBE = os.environ['GIT_DESCRIBE']
+TREE = os.environ['TREE']
+BRANCH = os.environ['BRANCH']
+ARCH = os.environ['ARCH']
+
+# Tokens
+LAVA_TBAKER_TOKEN = os.environ['LAVA_TBAKER_TOKEN']
+LAVA_MHART_TOKEN = os.environ['LAVA_MHART_TOKEN']
+LAVA_COLLABORA_TOKEN = os.environ['LAVA_COLLABORA_TOKEN']
+LAVA_BAYLIBRE_TOKEN = os.environ['LAVA_BAYLIBRE_TOKEN']
+LAVA_BAYLIBRE_SEATTLE_TOKEN = os.environ['LAVA_BAYLIBRE_SEATTLE_TOKEN']
+LAVA_FREE_ELECTRONS_TOKEN = os.environ['LAVA_FREE_ELECTRONS_TOKEN']
+
+# Command making the jobs
+make_jobs_args = [
+    'python',
+    'lava-v2-jobs-from-api.py',
+    '--callback %s' % CALLBACK,
+    '--api %s' % API,
+    '--storage %s' % STORAGE,
+    '--lab %s' % LAB,
+    '--describe %s' GIT_DESCRIBE,
+    '--tree %s' % TREE,
+    '--branch %s' % BRANCH,
+    '--arch %s' % ARCH,
+    '--plans boot boot-be boot-kvm boot-kvm-uefi boot-nfs boot-nfs-mp',
+    '--token %s' % API_TOKEN
+]
+# Command sending the jobs
+send_jobs_args = [
+    'python',
+    'lava-v2-submit-jobs.py',
+    '--username kernelci',
+    '--jobs %s' % LAB,
+]
+
+if LAB == "lab-tbaker" or LAB == "lab-tbaker-dev":
```

Comment:
Using a dictionary with the varying data and a single block of code would be easier to maintain.